config/routes.rb

```diff
@@ -11,15 +11,26 @@
     end
   end
 
-  resources :organizations, only: [:index, :show, :edit, :create, :update, :new]
+  resources :organizations, only: [:index, :show, :edit, :create, :update, :new] do
+    resources :memberships do
```

Comment:
Make sure to include an `only` clause on memberships, because we don't want people to be accessing `destroy`, `edit`, etc. controller actions.